index.html

```diff
@@ -1,63 +1,205 @@
+        </ul>
+        <a class="button" href="#">See project</a>
+      </div>
+    </div>
+    <div class="work-card">
+      <img class="work-snapshot element" src="./visuals/multi-post-stories-snapshot.png" />
+      <div class="work-text-block element">
+        <h1 class="title" id="headline-title">
+          Multi-Post Stories
+        </h1>
+        <ul class="client-role-year">
+          <li class="work-client">CANOPY</li>
+          <span class="bullet">&#x2022</span>
+          <li class="work-role">Back End Dev</li>
+          <span class="bullet">&#x2022</span>
+          <li class="work-year">2015</li>
+        </ul>
+        <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="tag-list">
+          <li class="tag">HTML</li>
+          <li class="tag">CSS</li>
+          <li class="tag">JavaScript</li>
+        </ul>
+        <a class="button" href="#">See project</a>
+      </div>
+    </div>
+    <div class="work-card">
+      <img class="work-snapshot element" src="./visuals/tonic-snapshot-02.png" />
+      <div class="work-text-block element">
+        <h1 class="title" id="headline-title">
+          Tonic
+        </h1>
+        <ul class="client-role-year">
+          <li class="work-client">CANOPY</li>
+          <span class="bullet">&#x2022</span>
+          <li class="work-role">Back End Dev</li>
+          <span class="bullet">&#x2022</span>
+          <li class="work-year">2015</li>
+        </ul>
+        <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="tag-list">
+          <li class="tag">HTML</li>
+          <li class="tag">CSS</li>
+          <li class="tag">JavaScript</li>
+        </ul>
+        <a class="button" href="#">See project</a>
+      </div>
+    </div>
+    <div class="work-card">
+      <img class="work-snapshot element" src="./visuals/multi-post-stories-snapshot-02.png" />
+      <div class="work-text-block element">
+        <h1 class="title" id="headline-title">
+          Multi-Post Stories
+        </h1>
+        <ul class="client-role-year">
+          <li class="work-client">CANOPY</li>
+          <span class="bullet">&#x2022</span>
+          <li class="work-role">Back End Dev</li>
+          <span class="bullet">&#x2022</span>
+          <li class="work-year">2015</li>
+        </ul>
+        <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="tag-list">
+          <li class="tag">HTML</li>
+          <li class="tag">CSS</li>
+          <li class="tag">JavaScript</li>
+        </ul>
+        <a class="button" href="#">See project</a>
       </div>
-    </section>
-  </body>
+    </div>
+  </section>
+  <section id="about-myself">
+    <div id="about-myself-container">
+      <h1 class="title" id="headline-title">
+        About Myself
+      </h1>
```

Comment:
✅ Please consider using a heading tag other than the h1 tag here as you have already used it in the headline section. The reason is there must be only one h1 tag through so the browser can easily understand what you want to show users when they search.